docs/reference/changelog.md

```diff
@@ -38,6 +38,7 @@
   * milestoneType
   * milestoneStatus
   * milestoneCode
+* [#429](https://github.com/open-contracting/infrastructure/pull/429) - add climateFinanceFocalPoint code to partyRole codelist.
```

Comment:
Since we're going to be adding lots of codes, we may as well organise them under a single changelog entry.

```suggestion
* Add codes:
  * partyRole:
    * [#429](https://github.com/open-contracting/infrastructure/pull/429) - 'climateFinanceFocalPoint'
```